### PR TITLE
 MAM-2930-make-a-new-endpoint-for-a-list-of-grouped-onboarding-accounts

### PR DESCRIPTION
**Pull request by jtomchak**
* Additional endpoint `/api/v2/onboarding_follow_recommendations/accounts` that will return only recommended accounts as an array. no hashtags or meta data

Try it out: https://staging.moth.social//api/v2/onboarding_follow_recommendations/accounts

Resolves MAM-2930

### CODE REVIEW

File: app/lib/onboarding/v2/onboarding_categories_development.yml [hunk 1]
```diff
@@ -27,3 +275,6 @@
     - account: '@jtomchak@moth.social'
       type: account
       summary: Engineer at Mammoth.
+    - account: '@pxl@moth.social'
+      type: account
+      summary: Designer at Mammoth.
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_development.yml:280]: I don't want to add pxl unless we have confirmation from him. (I know Bart asked, and I think he said "not yet", or… something).

**Review by rileyhCode — CHANGES_REQUESTED** [review]
One small change wrt @pxl

**Review by rileyhCode — APPROVED** [review]
Approved then.

File: app/lib/onboarding/v2/onboarding_categories_development.yml (hunk 1 above)
Comment (jtomchak) [reply app/lib/onboarding/v2/onboarding_categories_development.yml:280]: only for dev 👍

**Pull request merged**